src/colornames.csv

```diff
@@ -8864,6 +8883,7 @@ Escape Grey,#abac9f,
 Escargot,#fff1d8,x
 Escarpment,#d5b79b,
 Eshin Grey,#4a4f52,
+Eskimo Boot,#e9e5dd,

```

Comment:
That one is racist. Lets remove it: https://en.wikipedia.org/wiki/Eskimo#:~:text=In%20Canada%20and%20Greenland%2C%20and,a%20particular%20group%20or%20community.